cmd_states/run_test.py

```diff
@@ -0,0 +1,2 @@
+import Chandra.cmd_states
+Chandra.cmd_states.test()
```

Comment:
Right, regarding SKA I also wasn't sure if we needed a convention.

Regarding cond-build running tests by default, there is really no "default".  I just started the convention to add a `run_test.py` file that would run module tests if they seemed doable.  The big advantage I saw to doing tests from the package build process is the fact that the tests are run in the environment defined by the recipe requirements.  That means the tests are a great way to see if the recipe requirements/dependencies are really correct (assuming some appropriate level of tests coverage). For other kinds of system/regression/environment tests, obviously, it makes more sense to run ska_testr in the full/built environment.

Regarding test conventions and skipping; the conda recipe can run basically anything in a run_test.py or run_test.sh, so we could customize the called tests there.  Or skip altogether by not defining a run_test*...  or we could update the module tests to skip those tests from that side.

Also, right now there are no tests run on the modules that are *dependencies* of `testr` (Ska.Shell, Ska.File, pyyaks, ska_path?), as the modules wanted `testr` to run their tests, it hadn't been build yet, and I didn't see a great way to bootstrap that.